siwe_auth/backend.py

```diff
@@ -142,14 +143,12 @@ class ENSProfile:
     Container for ENS profile information including but not limited to primary name and avatar.
     """
 
-    name: str
-    avatar: str
+    name: str = ""
```

Comment:
Let's make these both default to `None`. The current behavior is that `ENS.fromWeb3(w3).name(address=ethereum_address)` will return `None` if a given request is not found.